### ISSUE TITLE
Mouse not recognized using BackSID

### ISSUE THREAD

**Issue by doughamm** [emanuelelaface/USBtoC64#2]
Using a BackSID, the pots won't register. BackSID is one of the few SID replacements that emulates pots but others that do may have problems as well.  BackSID works with MouSTer just fine but not USBtoC64.

**Comment by emanuelelaface** [comment]
Hi, at the time I developed this device I had a C64 with an original SID that produces the timing signal for the POTS all the time, and I created the project around that. Then a user reported the issue with ARMSID and I discovered that those clones do not generate the signal unless there is some voltage applied to the pot. So I fixed the design but I still have few adaptors from the previous design that are compatible only with the original SID. This is why on the site where I sell them I ask specifically to say if the user wants to use with a clone SID in order to send a modified version that is compatible with it.

By the way, I tested it with ARMSID and SIDKick, not with BackSID (I do not have one) but I guess it will work also with it because I think the only problem is to enable the digital generation of the signal on the POT pins.

Said this, please contact me on the place where you bought the adapter and I will try to fix this issue for you.

**Comment by doughamm** [comment]
Ah, I totally see now this option on the Tindy site that either didn't exist or I hadn't noticed back in September when I ordered! 

https://www.tindie.com/products/burglar_ot/usbtoc64/

About half my systems are BackSID while the rest are authentic; does the revised board retain compatibility with authentic SIDs or do you essentially need a board for each scenario? 

In any case I would love to acquire the revised one. Happy to do so however is most convenient. 

Thank you!

**Comment by emanuelelaface** [comment]
Yes the other revision works with every system, but maybe you can modify your without order a new one. I will write you from Whatsapp if you don't mind, on the number provided on Tindie.

**Comment by doughamm** [comment]
Just updating the issues log to echo my WhatsApp discussion for the greater community: adding a resistor gets it to recognize a mouse on the BackSID, and it does move, but it's unusably jumpy (whether a mouse is connected or not).  I've also let Evie know on her forum since this may be a firmware-solvable issue as much as a hardware one: https://forum.backbit.io/index.php?topic=1218.0

**Comment by emanuelelaface** [comment]
I will close here. I think the issue with BackSID is possibly due to the voltage expected buy the replacement SID or something else. I cannot test it in my lab because I do not have one and to do a full debug I need to see the signals with an oscilloscope. I had the possibility to test it with ARMSID and SIDKick, both work fine, but BackSID probably needs +5v as HIGH signal while the ESP32 board uses +3.3v.
